pkg/operator/starter.go

```diff
@@ -79,6 +80,7 @@ func RunOperator(ctx context.Context, cc *controllercmd.ControllerContext) error
 		discoveryClient,
 		crdClient,
 		cc.EventRecorder,
+		os.Getenv("RELATED_IMAGE_OPERAND_IMAGE"),
```

Comment:
I'm probably missing something but it's not clear to me who is supposed to set this env. set in the pod spec of the operator?

Reply:
Correct.

Reply:
Is this clear now?

Reply:
yup all good!